### PR TITLE
Add support for mocking referrer in Request

### PR DESCRIPTION
**Pull request by sreedhar7**

### CODE REVIEW

File: packages/service-worker-mock/models/Request.js [hunk 1]
```diff
@@ -44,6 +45,7 @@ class Request extends Body {
 
     this.method = options.method || 'GET';
     this.mode = options.mode || 'same-origin';   // FF defaults to cors
+    this.referrer = options.referrer || undefined;
```
Comment (zackargyle) [inline packages/service-worker-mock/models/Request.js]: The spec says that if the referrer is `no-referrer` it should return an empty string. Can we change this line to
```
this.referrer = options.referrer && options.referrer !== 'no-referrer' ? options.referrer : '';
```

Comment (sreedhar7) [reply packages/service-worker-mock/models/Request.js]: done

**Comment by zackargyle** [comment]
@jparise is there anyone at Pinterest who would like to review as well?

**Comment by jparise** [comment]
@bishwei would be a good person.

**Comment by sreedhar7** [comment]
Thank you @zackargyle for approving. I really appreciate if we can merge this PR soon and publish a new version of he package.

**Comment by zackargyle** [comment]
@sreedhar7 I no longer have admin rights to merge, but I've notified @bishwei and he said he would try to get to it later today. I'll make sure it gets published to NPM shortly after.

**Comment by sreedhar7** [comment]
@bishwei I really appreciate if you can take a look at this PR and merge it today.

**Comment by bishwei** [comment]
@sreedhar7 working on it right now, sorry for the delay

**Comment by zackargyle** [comment]
@sreedhar7 Just as a note, the last release was a fairly significant backwards-incompatible change to make it possible for Google's Workbox project to use it better. If you're not on 2.0.0 already, you will probably want to take a look at the pull request (#109) from @kommander.

**Comment by sreedhar7** [comment]
Thanks for the note @zackargyle. We started using the package recently and are already on 2.0.0. Please let us know when the new build with the fix has been published.

**Comment by bishwei** [comment]
@sreedhar7 published 2.0.3 which includes your change.  Thanks again for the contribution!

**Comment by sreedhar7** [comment]
Thank you @bishwei and @zackargyle for your quick responses.

**Pull request merged**